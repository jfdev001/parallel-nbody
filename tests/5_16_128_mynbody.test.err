Reservation number 3909812: Reserved 5 hosts for 900 seconds 
Run on 5 hosts for 960 seconds from Sun Jan 15 15:37:12 2023
: node059/0 node059/1 node059/2 node059/3 node059/4 node059/5 node059/6 node059/7 node059/8 node059/9 node059/10 node059/11 node059/12 node059/13 node059/14 node059/15 node060/0 node060/1 node060/2 node060/3 node060/4 node060/5 node060/6 node060/7 node060/8 node060/9 node060/10 node060/11 node060/12 node060/13 node060/14 node060/15 node061/0 node061/1 node061/2 node061/3 node061/4 node061/5 node061/6 node061/7 node061/8 node061/9 node061/10 node061/11 node061/12 node061/13 node061/14 node061/15 node062/0 node062/1 node062/2 node062/3 node062/4 node062/5 node062/6 node062/7 node062/8 node062/9 node062/10 node062/11 node062/12 node062/13 node062/14 node062/15 node063/0 node063/1 node063/2 node063/3 node063/4 node063/5 node063/6 node063/7 node063/8 node063/9 node063/10 node063/11 node063/12 node063/13 node063/14 node063/15 
Running N-body with 128 bodies and 1000 steps
