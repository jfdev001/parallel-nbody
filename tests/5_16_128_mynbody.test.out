[node059:31040] *** An error occurred in MPI_Gather
[node059:31040] *** reported by process [1168703489,0]
[node059:31040] *** on communicator MPI_COMM_WORLD
[node059:31040] *** MPI_ERR_TRUNCATE: message truncated
[node059:31040] *** MPI_ERRORS_ARE_FATAL (processes in this communicator will now abort,
[node059:31040] ***    and potentially your MPI job)
STATUS=PASSED
